integration-tests/tls_test.go

```diff
@@ -1056,16 +1057,36 @@ func testProxyClientTls(t *testing.T, ccmSetup *setup.CcmTestSetup,
 
 	logMessages := buffer.String()
 
+	warnCheckFail := false
+	warnCheckDone := false
```

Comment:
The `Done` suffix here confused me for a second, I thought it would be set when the result of some async processing was done, but in reality this value is simply set when a warning is expected. My suggestion would be to name this `warningExpected` instead. The same goes for `errCheckDone` below, `errExpected`.

Reply:
done